get_location.py

```diff
@@ -0,0 +1,53 @@
+"""Grab location from API based on users input. Currently set up to search for city names based on input only in the United States."""
+""" Currently when looking up by state only returns the state instead of all cities in the state"""
+import requests
+
+
+params = {
+    'city': '',
+    'country': 'United States',
+    'format': 'json'
+}
+
+def get_location(userInput):
+    """Gather all methods into one in order to create location object. 
+    Only needs to call this function to excecute all of them."""
+    try:
+        create_params(userInput)
+        returned_data = request_nominatim()
+        format_data(returned_data)
+    except requests.exceptions.HTTPError as e:
+        print("An HTTP error has occurred.", e)
+    except requests.exceptions.RequestException as e:
+        print("An error has occurred.", e)
+
+def create_params(userInput):
+    """ Creates the search string for the api to search"""
+    if userInput.isdigit(): # Prevents number input
+        raise ValueError("Input must be a city name.")
+    userInput = userInput
+    searchString = f'{userInput}'
+    params.update({'city': searchString})
+    pass
```

Comment:
`pass` is unnecessary here. `pass` is usually placeholder for doing nothing, but the function already does stuff.

Reply:
I agree it was a placeholder, but I forgot to get rid of it. There is a new version of the file that resolves this issue.